content/desktop/install/windows-install.md

```diff
@@ -93,6 +90,22 @@ For more information on setting up WSL 2 with Docker Desktop, see [WSL](../wsl/_
 >
 > Docker only supports Docker Desktop on Windows for those versions of Windows that are still within [Microsoft’s servicing timeline](https://support.microsoft.com/en-us/help/13853/windows-lifecycle-fact-sheet). Docker Desktop is not supported on server versions of Windows, such as Windows Server 2019 or Windows Server 2022. For more information on how to run containers on Windows Server, see [Microsoft's official documentation](https://learn.microsoft.com/virtualization/windowscontainers/quick-start/set-up-environment).
 
+{{< /tab >}}
+{{< tab name="Windows with Apple silicon (Beta)" >}}
+
+- Windows 11 64-bit: Pro version 21H2 or higher, or Enterprise or Education version 21H2 or higher.
+- Windows 10 64-bit:
+  - We recommend Home or Pro 22H2 (build 19045) or higher, or Enterprise or Education 22H2 (build 19045) or higher. 
+  - Minimum required is Home or Pro 21H2 (build 19044) or higher, or Enterprise or Education 21H2 (build 19044) or higher.
+
+> **Important**
+>
+> The following features are not supported:
+> - Hyper-V backend
+> - Windows containers
+> - The installer and the privileged service
```

Comment:
The installer and the privileged service are supported, I think we just need to add a note that they x64 application which run under emulation on ARM.
cc @doringeman for the correct terminology